build-system/test-configs/karma.conf.js

```diff
@@ -80,6 +80,10 @@ module.exports = {
     },
   },
 
+  junitReporter: {
+    useBrowserName: false,
+  },
```

Comment:
It seems weird to set one sub-option statically (here) and the others dynamically (in `runtime-test-base.js`). Given that this is only applicable on CI, I think it's safe to move this to the dynamic section as well.